prow/plugins.yaml

```diff
@@ -73,6 +73,14 @@ approve:
   implicit_self_approve: true
   lgtm_acts_as_approve: true
 
+# Lower bounds in number of lines changed; XS is assumed to be zero.
```

Comment:
might be worth mentioning that there is an attempt to exclude certain kinds of generated lines 
(which also should perhaps be more configurable...)

Reply:
@BenTheElder It's on my radar to add support for looking in `.gitattributes` for vendored/generated directives as it's recognised by GitHub already for identifying generated code (we're currently duplicating the information between `.generated_files` for this plugin and `.gitattributes` for GitHub) -- is that something which would be welcome?

Reply:
I'd guess so, I have no idea who set up the initial "ignore generated files", I've just run into it before when a huge PR got marked size/s :joy: